packages/preferences/src/common/json-validator.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export const JsonValidator = Symbol("JsonValidator");
+
+export interface JsonValidator {
```

Comment:
Why not just call `ajv.validate(schema, data)` instead of introducing `JsonValidator` and `JsonPreferenceValidator`. It is one method that does everything.